package.json

```diff
@@ -9,16 +9,16 @@
   "scripts": {
     "check-ci": "npm-run-all test-with-coverage build-integration-test test-blockchain-with-coverage coverage lint",
     "check": "npm-run-all test-with-coverage integration-test-ganache coverage lint",
-    "eslint": "eslint --max-warnings=0 src test",
-    "lint-autofix": "eslint --fix src test",
+    "eslint": "eslint --max-warnings=0 src test config",
```

Comment:
can we put config inside src? Is there any reason for leaving it out?

Reply:
fixed